build/generate_bundle.sh

```diff
@@ -34,6 +34,18 @@ generate_bundle() {
     popd > /dev/null 2>&1
 
     sed -i -E "${REPLACE_REGEX}" "${WORKING_DIR}/manifests/${OPERATOR_NAME}.clusterserviceversion.yaml"
+    cat >> "${WORKING_DIR}/metadata/annotations.yaml" << EOF
+  features.operators.openshift.io/disconnected: "false"
+  features.operators.openshift.io/fips-compliant: "false"
```

Comment:
My hunch is that we could run in a FIPS-140 environment, but I don't think we have any place to do explicit testing yet. Can shiftstack deploy OCP in fips mode? This might be trivial to try.

Reply:
Yea we have this spike (https://issues.redhat.com/browse/STF-1384) for STF + FIPS filed a long time ago. I think we were mostly waiting for our CI framework to get updated and make it easier to add different deployment environments.

This might be something we could look at doing as part of disconnected because we'll want our CI systems to deploy OCP in disconnected mode, and then work through installation of STF via disconnected mode as our default installation method. If it's relatively simple to put OCP into FIPS compatible mode as well, then we can layer FIPS + disconnected in the same environment.